test/jdk/java/awt/Mixing/AWT_Mixing/MixingPanelsResizing.java

```diff
@@ -313,6 +343,21 @@ public static synchronized void fail(String whyFailed) {
         mainThread.interrupt();
     }//fail()
 
+    private static boolean isAlmostEqualColor(Color color, Color refColor) {
+
+        System.out.println("Comparing color: " + color + " with reference " +
+                "color: " + refColor);
+        return color.equals(refColor) ||
+                Math.abs(color.getRed() - refColor.getRed()) <
+                        TOLERANCE_MACOSX &&
+                        Math.abs(color.getGreen() - refColor.getGreen()) <
+                                TOLERANCE_MACOSX &&
+                        Math.abs(color.getBlue() - refColor.getBlue()) <
+                                TOLERANCE_MACOSX;
+    }
+
     static class TestPassedException extends RuntimeException {
+
     }
```

Comment:
I see no reason to add a blank line inside the declaration of the `TestPassedException` class.